src/Account.sol

```diff
@@ -383,16 +392,16 @@ contract Account is IAccount, Auth, OpsReady {
                         _desiredFillPrice: desiredFillPrice,
                         _reduceOnly: reduceOnly
                     });
-                } else if (_command == Command.GELATO_CANCEL_CONDITIONAL_ORDER)
-                {
+                } else {
+                    // Command.GELATO_CANCEL_CONDITIONAL_ORDER
                     uint256 orderId;
                     assembly {
                         orderId := calldataload(_inputs.offset)
                     }
                     _cancelConditionalOrder({_conditionalOrderId: orderId});
-                } else {
-                    revert InvalidCommandType(commandIndex);
                 }
+            } else {
+                revert InvalidCommandType(commandIndex);
```

Comment:
**Note:** Codecov is telling me this line is not covered by tests.

Reply:
I noticed this too but it is tested [here](https://github.com/Kwenta/margin-manager/blob/be54d21d0b74b80cfdea0313bd15f1a1439993c7/test/unit/Account.t.sol#L346) in `Account.t.sol`.

`forge coverage` is much better than it was but still not perfect. Used to be highly inaccurate actually not that long ago. 

Good attention to detail here though 👍